packages/design-tokens/src/tokens/functional/components/anchor-nav/colors.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  brand: {
+    AnchorNav: {
+      border: {
+        nav: {
+          default: {
```

Comment:
@langermank - based on the recent token ADR, I'm thinking of changing these tokens to:

- `--brand-anchornav-link-underline-hover`
- `--brand-anchornav-link-underline-pressed`

Does that seem reasonable?